packages/node/src/models/proposed-app-instance-info.ts

```diff
@@ -63,16 +63,20 @@ export class ProposedAppInstanceInfo implements AppInstanceInfo {
     appInstanceId: AppInstanceID,
     proposeParams: IProposedAppInstanceInfo
   ) {
+    if (!proposeParams.abiEncodings.actionEncoding) {
+      delete proposeParams.abiEncodings.actionEncoding;
+    }
```

Comment:
We can just JSON stringify and and JSON parse at firebase-entry-time to handle this I think

Reply:
Done.